python/lsst/cp/pipe/plotPtc.py

```diff
@@ -810,50 +817,63 @@ def _plotStandardPtc(self, dataset, ptcFitType, pdfPages):
+            a2.set_xlabel(r'Mean Signal ($\mu$, DN)', fontsize=labelFontSize)
+            a2.set_ylabel(r'Variance (DN$^2$)', fontsize=labelFontSize)
+            a2.tick_params(labelsize=11)
+            a2.set_xscale('log')
+            a2.set_yscale('log')
+
+            a3.set_xlabel(r'Mean signal ($\mu$, DN)', fontsize=labelFontSize)
+            a3.set_ylabel(r'Variance/$\mu$ (DN)', fontsize=labelFontSize)
+            a3.tick_params(labelsize=11)
+            a3.set_xscale('linear', fontsize=labelFontSize)
+            a3.set_yscale('linear', fontsize=labelFontSize)
+
+            minMeanVecFinal = np.min(meanVecFinal)
+            maxMeanVecFinal = np.max(meanVecFinal)
+            meanVecFit = np.linspace(minMeanVecFinal, maxMeanVecFinal, 100*len(meanVecFinal))
+            minMeanVecOriginal = np.min(meanVecOriginal)
+            maxMeanVecOriginal = np.max(meanVecOriginal)
+            deltaXlim = maxMeanVecOriginal - minMeanVecOriginal
+
+            a.plot(meanVecFit, ptcFunc(pars, meanVecFit), color='red')
+            a.plot(meanVecFinal, ptcNoiseAdu**2 + (1./ptcGain)*meanVecFinal, color='green',
+                   linestyle='--')
+            a.scatter(meanVecFinal, varVecFinal, c='blue', marker='o', s=markerSize)
+            a.scatter(meanVecOutliers, varVecOutliers, c='magenta', marker='s', s=markerSize)
+            a.text(0.03, 0.7, stringLegend, transform=a.transAxes, fontsize=legendFontSize)
+            a.set_title(amp, fontsize=titleFontSize)
+            a.set_xlim([minMeanVecOriginal - 0.2*deltaXlim, maxMeanVecOriginal + 0.2*deltaXlim])
+
+            # Same, but in log-scale
+            a2.plot(meanVecFit, ptcFunc(pars, meanVecFit), color='red')
+            a2.scatter(meanVecFinal, varVecFinal, c='blue', marker='o', s=markerSize)
+            a2.scatter(meanVecOutliers, varVecOutliers, c='magenta', marker='s', s=markerSize)
+            a2.text(0.03, 0.7, stringLegend, transform=a2.transAxes, fontsize=legendFontSize)
+            a2.set_title(amp, fontsize=titleFontSize)
+            a2.set_xlim([minMeanVecOriginal, maxMeanVecOriginal])
+
+            # Var/mu vs mu
+            a3.plot(meanVecFit, ptcFunc(pars, meanVecFit)/meanVecFit, color='red')
+            a3.scatter(meanVecFinal, varVecFinal/meanVecFinal, c='blue', marker='o', s=markerSize)
+            a3.scatter(meanVecOutliers, varVecOutliers/meanVecOutliers, c='magenta', marker='s',
+                       s=markerSize)
+            a3.text(0.2, 0.65, stringLegend, transform=a3.transAxes, fontsize=legendFontSize)
+            a3.set_title(amp, fontsize=titleFontSize)
+            a3.set_xlim([minMeanVecOriginal - 0.2*deltaXlim, maxMeanVecOriginal + 0.2*deltaXlim])
 
         f.suptitle("PTC \n Fit: " + stringTitle, fontsize=supTitleFontSize)
         pdfPages.savefig(f)
         f2.suptitle("PTC (log-log)", fontsize=supTitleFontSize)
         pdfPages.savefig(f2)
+        f3.suptitle("Var/$mu$", fontsize=supTitleFontSize)
```

Comment:
This needs a backslash I believe to properly format the μ.

Reply:
Can't you just use the unicode character directly?

Reply:
The rest of the code uses LaTeX style, so I think it's better to be consistent.